analysers/experiments/cpu.py

```diff
@@ -181,6 +181,12 @@ def sortAndGet(field, asc):
 CassandraRDD = sc.cassandraTable(cassandraKeyspace, "trial_cpu_core")
 CassandraRDD.cache()
 
+nOfCores = CassandraRDD.select("cpu_mean") \
```

Comment:
Why do you need to access the `nOfCores` this way? We should have it in the Cassandra schema already.